### DISCUSSION TITLE
Duplicate Request header "Accept" : "application/problem+json" added in postman while uploading the collection

### DISCUSSION THREAD

**Discussion by disha-shah1** [apideck-libraries/portman#450] [Q&A]
Duplicate Request header `"Accept" : "application/problem+json" `added in postman while uploading the collection. this fails my test.

when i upload collection through portman, this duplicate accept header gets added, while Accept header is already available there as `"Accept": "*/*"` in postman, i have tried below which does not work.

```
"overwriteRequestHeaders ": [
                  {
                    "key": "Accept",
                    "value": "application/problem+json",
                    "remove": true
                  }
                ]
```

![image](https://user-images.githubusercontent.com/62923897/213986319-9e05abfa-2197-41c8-9b91-d25467704969.png)

**Comment by thim81** [comment]
hi @disha-shah1 

This seems like a bug.
We will investigate it and circle back to you.
